lib/addons/eks-managed-core-addons/index.ts

```diff
@@ -0,0 +1,30 @@
+import { CfnAddon } from "@aws-cdk/aws-eks";
```

Comment:
`CfnAddon` -> `CfnAddOn` everywhere.

Reply:
HI @askulkarni2 , I noticed that the CfnAddon Library doesn't have a `O`. Same goes for its properties https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-eks.CfnAddon.html